src/components/footer/footer.js

```diff
@@ -1,7 +1,48 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Link from '@material-ui/core/Link';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
```

Comment:
if you want you can write it as `import { makeStyles, Grid, Typography, Link, Container } from '@material-ui/core/';`